### ISSUE TITLE
Hide advanced search from screen readers: 0.5

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#250]
The ISAW website visual design hides the "advanced search" option from sighted users, but it still appears to users of screen readers. Make behavior consistent.

> Assistive technology API’s and users depend on proper state management to properly navigate and convey page structure. Off-screen elements that are improperly hidden are still accessible to keyboard and screen reader users, which can cause issues with navigation and adds keystrokes to user’s navigation. Hiding content using CSS styles display:none and visibility:hidden endure that it is not available to assistive technology users when not visible.
> - While using a screen reader, the search bar provides the option for an advanced search that is not visible to the user.

Traceability: [Microassist Report 1 > WCAG Compliance > [ no section title -- comes after focus indication ]](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.2i8dt2er2lqq)

**Comment by paregorios** [comment]
So, add a ```visibility: hidden``` directive to the CSS for ```#portal-searchbox, so in ```theme.css``` where we currently have:

```css
#portal-searchbox {
	display: none;
}
```

we want to have:

```css
#portal-searchbox {
	display: none;
        visibility:hidden;
}

**Comment by alecpm** [comment]
This is live on staging

**Comment by paregorios** [comment]
This looks good. Please merge to master and deploy to production.

For future reference, we decided in the end to nuke the advanced search affordance from orbit using a Diazo rule, rather than fooling around with CSS as suggested above.
